CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [4.5.5]
+
+- update ring-middleware to 4.5.5, which adds
```

Comment:
it looks like ring-middleware is being updated to "1.1.0"?

Reply:
Woops, yeah. Accidentally put the clj-parent version. I'll push a fix.